exercises/ocr-numbers/ocr-numbers.example.ts

```diff
@@ -0,0 +1,92 @@
+interface Patterns {
+    [key: number]: string[]
+}
+
+const PATTERNS: Patterns = {
+    0: [' _ ',
+        '| |',
+        '|_|',
+        '   '],
+    1: ['   ',
+        '  |',
+        '  |',
+        '   '],
+    2: [' _ ',
+        ' _|',
+        '|_ ',
+        '   '],
+    3: [' _ ',
+        ' _|',
+        ' _|',
+        '   '],
+    4: ['   ',
+        '|_|',
+        '  |',
+        '   '],
+    5: [' _ ',
+        '|_ ',
+        ' _|',
+        '   '],
+    6: [' _ ',
+        '|_ ',
+        '|_|',
+        '   '],
+    7: [' _ ',
+        '  |',
+        '  |',
+        '   '],
+    8: [' _ ',
+        '|_|',
+        '|_|',
+        '   '],
+    9: [' _ ',
+        '|_|',
+        ' _|',
+        '   '],
+}
+
+export default class OcrParser {
+    convert(text: string) {
```

Comment:
Why not just make this static tool?

Reply:
I've made the change in the latest commit